### PR TITLE
Feat add testing to pipeline

### PR DESCRIPTION
**Pull request by Trishisingh**

### CODE REVIEW

File: de_build/adf/adf-pipeline.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ pool:
 
 variables:
   - template: adf-pipeline-vars.yml
+  - template: air-data-testing-vars.yml
```
Comment (george-calvert-ensono) [inline de_build/adf/adf-pipeline.yml:30]: What does air mean?

Comment (george-calvert-ensono) [reply de_build/adf/adf-pipeline.yml:30]: Ah assuming it's Airflow. Need to check what we're using and remove unused vars. Also need to rename the file

File: de_build/adf/air-data-testing-vars.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+variables:
+  - name: pythonVersion
+    value: '3.10.10'
+  - name: airflowBucket
+    value: ''
+  - name: servicePath
+    value: 'dags/'
```
Comment (george-calvert-ensono) [inline de_build/adf/air-data-testing-vars.yml:5]: there's no dags folder

File: de_build/adf/air-data-testing-vars.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - name: requirementsPath
+    value: dags/requirements.txt
```
Comment (george-calvert-ensono) [inline de_build/adf/air-data-testing-vars.yml:7]: no requirements folder

**Review by george-calvert-ensono — CHANGES_REQUESTED** [review]

File: de_build/adf/adf-pipeline.yml (hunk 1 above)
Comment (Trishisingh) [reply de_build/adf/adf-pipeline.yml:30]: air mean ( amido independent runnner)

File: de_build/adf/air-data-testing-vars.yml (hunk 3 above)
Comment (Trishisingh) [reply de_build/adf/air-data-testing-vars.yml]: thanks. removed

File: de_build/adf/air-data-testing-vars.yml (hunk 2 above)
Comment (Trishisingh) [reply de_build/adf/air-data-testing-vars.yml]: removed

**Review by satenderrathee — APPROVED** [review]

**Review by george-calvert-ensono — APPROVED** [review]

**Pull request merged**